### ISSUE TITLE
EL24E RX sends 122.9% stick range over CRSF

### ISSUE THREAD

**Issue by t413** [ExpressLRS/ExpressLRS#1396]
## Current Behavior

[This rx](https://www.racedayquads.com/products/flywoo-elrs-el24e-2-4-ghz-receiver-w-ceramic-antenna), the Flywoo EL24E, tries to send values in excess of 885 through 2114 when it should be sending 1000 - 2000. Betaflight shows sticks working as normal, just scaled incorrectly _past_ the range either Betaflight can understand or CRSF can transmit. Centering is correct around 1500. 

The _same_ model and config on my tango 2, transmitting to an AIO ELRS board, correctly reports a perfect 1000 - 2000 range (I can test it that way since model match is turned off)

Massively scaling the TX outputs range down makes things successfully flyable.

## Steps to Reproduce

1. Connect AIO receiver, observe correct results in the receivers page of BF.
2. Connect EL24E instead, observe incorrectly scaled stick ranges in BF.

## Details

I used a logic analyzer to intercepted the CRSF data from the receiver. Here's a captured packet:
`\xC8\x18\x16\xE0\x03\x1F0\xB8\xF7\x8B_\xFC\xE2\x17\xE0\x03\x1F\xF8(\x08\0\0D<\xE2\xE2\`

The full capture is [here on db](https://www.dropbox.com/s/hlkkev8rgr7eanx/csrf%20of%20EL24E%20bad%20range.sal?dl=0) as a Salae Logic file.

## Your Environment

- TX hardware: BetaFPV 2.4 micro TX
- RX hardware: Flywoo EL24E
- Handset model: Tango 2
- OpenTX version (including nightly number) TBS-1.3.0
- ExpressLRS version (TX & RX MUST MATCH): 2.2.0 on both. Also tested 2.0.1 and 2.1.0 on the RX as well.
- Packet Rate: 250 Hz
- Telemetry Ratio: 1:128
- user_defines:

**Comment by t413** [comment]
Doing some analysis– I've written a C++ program using some example code from the PX4 project to parse that captured CRSF data.

Here's [the program on a CPP playground](http://cpp.sh/6idbb) for future reference.

The CRSF data is perfectly correct, this is looking like a betaflight problem after all.

Closing, but I will update here when I figure out what's going wrong in BF.

**Comment by t413** [comment]
Gah. Figured it out.

Started digging around in the BF CLI and found `rxrange` command. The values were not changed from factory defaults, but they defaulted to:

```
rxrange 0 1159 1841
rxrange 1 1159 1841
rxrange 2 1159 1841
rxrange 3 1159 1841
```

This must be because it's an integrated SPI DSM receiver that I've disabled and use CRSF with instead. Geez.

Set them to `rxrange 0 1000 2000` and now everything works perfectly. 🤦🏼‍♂️
